Public/css/explore.css

```diff
@@ -101,7 +101,17 @@ footer {
     margin-right: 10px;
 }
 
+.accessory-details-container {
+    position: relative;
+    top: 20%
+}
+
 .coming-soon-icon {
     float: right;
     margin-bottom: 0px;
 }
+
+.bridge-icon {
+    float: right;
+    margin-bottom: 0px;
```

Comment:
Can you please add `margin-right: 5px;` so that we have some spacing between the icons?

Reply:
Yes, but it's valid for case when we have both icons. But what if we have only bridge icon, without second one? Is it ok to leave right margin? Not really strong in css =)

Reply:
Right, that makes sense. To fix that, since all icons are right-aligned, we can set `margin-left: 5px;` on both the bridge and coming-soon icons so if there's only one, the margin won't be visible.